tty-acs.c

```diff
@@ -146,19 +146,19 @@ static const struct utf8_data tty_acs_heavy_borders_list[] = {
 
 /* UTF-8 rounded borders. */
 static const struct utf8_data tty_acs_rounded_borders_list[] = {
-       { "", 0, 0, 0 },
-       { "\342\224\202", 0, 3, 1 }, /* U+2502 */
-       { "\342\224\200", 0, 3, 1 }, /* U+2500 */
-       { "\342\225\255", 0, 3, 1 }, /* U+256D */
-       { "\342\225\256", 0, 3, 1 }, /* U+256E */
-       { "\342\225\260", 0, 3, 1 }, /* U+2570 */
-       { "\342\225\257", 0, 3, 1 }, /* U+256F */
-       { "\342\224\263", 0, 3, 1 }, /* U+2533 */
-       { "\342\224\273", 0, 3, 1 }, /* U+253B */
-       { "\342\224\243", 0, 3, 1 }, /* U+2523 */
-       { "\342\224\253", 0, 3, 1 }, /* U+252B */
-       { "\342\225\213", 0, 3, 1 }, /* U+254B */
-       { "\302\267",     0, 2, 1 }  /* U+00B7 */
+	{ "", 0, 0, 0 },
+	{ "\342\224\202", 0, 3, 1 }, /* U+2502 */
+	{ "\342\224\200", 0, 3, 1 }, /* U+2500 */
+	{ "\342\225\255", 0, 3, 1 }, /* U+256D */
+	{ "\342\225\256", 0, 3, 1 }, /* U+256E */
+	{ "\342\225\260", 0, 3, 1 }, /* U+2570 */
+	{ "\342\225\257", 0, 3, 1 }, /* U+256F */
+	{ "\342\224\263", 0, 3, 1 }, /* U+2533 */
+	{ "\342\224\273", 0, 3, 1 }, /* U+253B */
+	{ "\342\224\243", 0, 3, 1 }, /* U+2523 */
+	{ "\342\224\253", 0, 3, 1 }, /* U+252B */
+	{ "\342\225\213", 0, 3, 1 }, /* U+254B */
+	{ "\302\267",     0, 2, 1 }  /* U+00B7 */
```

Comment:
I've been listening to ["The Style Hymn"](https://www.openbsd.org/lyrics.html#70) while reading style(9) :smile: